app/models/facts/metric.rb

```diff
@@ -21,7 +21,7 @@ def is_this_useful_no=(value)
     self.satisfaction_score = calculate_satisfaction_score
   end
 
-  scope :for_yesterday, -> { where(dimensions_date: Dimensions::Date.for(Date.yesterday)) }
+  scope :within_yesterday, -> { where(dimensions_date: Dimensions::Date.for(Date.yesterday)) }
```

Comment:
I think `.for_yesterday` made more sense to my reading, but I understand why you've renamed it. I don't think `.within_yesterday` reads very well though. Perhaps `.from_yesterday` covers your meaning?

Reply:
I will remove the commit as per our conversation.